docs/how-to-guides/azure_resource_provision.json

```diff
@@ -18,28 +18,47 @@
     },
     "allowAllConnections": {
       "defaultValue": "true",
-      "allowedValues": ["true", "false"],
+      "allowedValues": [ "true", "false" ],
       "type": "String",
       "metadata": {
         "description": "Specifies whether to allow client IPs to connect to Synapse"
       }
     },
     "provisionPurview": {
       "defaultValue": "true",
-      "allowedValues": ["true", "false"],
+      "allowedValues": [ "true", "false" ],
       "type": "String",
       "metadata": {
         "description": "Whether or not put purview in the provision script"
       }
     },
     "provisionEventHub": {
       "defaultValue": "true",
-      "allowedValues": ["true", "false"],
+      "allowedValues": [ "true", "false" ],
       "type": "String",
       "metadata": {
         "description": "Whether or not to deploy eventhub provision script"
       }
-    }
+    },
+    "dockerImage": {
+      "defaultValue": "blrchen/feathr-sql-registry",
```

Comment:
Are we going to have Blair's alias in the default value?

Reply:
Blair is going to change it and then I will update as part of a different PR